lib/manageiq/schema/engine.rb

```diff
@@ -3,6 +3,10 @@ module Schema
     class Engine < ::Rails::Engine
       isolate_namespace ManageIQ::Schema
 
+      # This auto-generated line is intentionally commented out to avoid
+      #   autoload of helper classes when part of ManageIQ core.
+      # config.autoload_paths << root.join('lib').to_s
```

Comment:
I want to get rid of this and use the default, but we have a number of classes in the lib directory that are actually more like helper classes when this plugin is standalone, and so need to be moved / renamed.